data/eos-metrics-event-recorder.service.in

```diff
@@ -9,6 +9,7 @@ User=metrics
 Type=dbus
 BusName=com.endlessm.Metrics
 ExecStart=@libexecdir@/eos-metrics-event-recorder
+ExecReload=/bin/kill -SIGHUP $MAINPID
```

Comment:
`man systemd.service` actually recommends against this approach – it suggests using a mechanism which allows you to wait for it to complete (and see if it was successful). An example from `dbus.service` itself:

```
/lib/systemd/system/dbus.service:ExecReload=/usr/bin/dbus-send --print-reply --system --type=method_call --dest=org.freedesktop.DBus / org.freedesktop.DBus.ReloadConfig
```

(I suggested the `SIGHUP` approach for `gdm` because it already implemented a `SIGHUP` handler.)

But I appreciate that this approach does not involve adding methods to the API (defined in a separate package) etc etc.

Reply:
I suppose we could add some new d-bus API to do this, it wouldn't be too hard.

At that point I might as well add some API to wipe the existing metrics cache too.